.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
+known_first_party = estuary
```

Comment:
Should this also include `scrapers` and `tests`?

Reply:
Why did you add estuary there then if it can auto detect?

Reply:
because it didn't auto detect